project/thscoreboard/replays/views/create_replay.py

```
@@ -61,6 +63,10 @@ def upload_file(request):
 
             except ValidationError as e:
                 form.add_error('replay_file', e)
+            except MemoryError as e:
```

It's generally considered not a good idea to catch MemoryError.

The reason for this is that if Python raises MemoryError, something very weird is going on in the server. It should never happen under normal circumstances, no matter what is uploaded (there's a reason we limit the size of replays, after all). As such, we should let the exception bubble up to Django. That way, it will result in a 500 error. When we add internal monitoring later (with something like Datadog or Honeycomb, or built ourselves), we can track all 500s and know there is a bug when one happens.

(At some point, we should add a generic 500 error page, as mentioned in https://docs.djangoproject.com/en/4.1/ref/views/#error-views.)